spec.html

```diff
@@ -27881,8 +28136,8 @@ <h1>
         </dl>
 
         <emu-alg>
-          1. Assert: Exactly one element of _referrer_.[[LoadedModules]] is a Record whose [[Specifier]] is _specifier_, since LoadRequestedModules has completed successfully on _referrer_ prior to invoking this abstract operation.
-          1. Let _record_ be the Record in _referrer_.[[LoadedModules]] whose [[Specifier]] is _specifier_.
+          1. Assert: Exactly one element of _referrer_.[[LoadedModules]] is a LoadedModuleRequest Record _record_ such that ModuleRequestsEqual(_record_, _request_) is *true*, since LoadRequestedModules has completed successfully on _referrer_ prior to invoking this abstract operation.
+          1. Let _record_ be the LoadedModuleRequest Record in _referrer_.[[LoadedModules]] such that ModuleRequestsEqual(_record_, _request_) is *true*.
```

Comment:
The form of this `Let` step is unusual. Normally, in `Let _foo_ be <something>`, the alias `_foo_` doesn't appear in `<something>`.

For the 2 steps, maybe something more like:
```
1. Let _x_ be a List consisting of
     each LoadedModuleRequest Record _record_
     of _referrer_.[[LoadedModules]] such that etc.
3. Assert: _x_ has exactly one element.
4. Let _record_ be the sole element of _x_.
```
(That also eliminates the duplication between the two steps.)

Reply:
Uh ecmarkup's linting doesn't like this

Reply:
Yeah, given
```
... a List consisting of each LoadedModuleRequest Record _r_
    of _referrer_.[[LoadedModules]] such that <condition>
```
ecmarkup is saying that it can't find a preceding declaration for `_r_`. That is, it's treating this occurrence of `_r_` as a 'use', whereas it's syntactically a declaration.

@bakkot: could ecmarkup be modified to treat this as a declaration?